solr/core/src/java/org/apache/solr/search/SolrIndexSearcher.java

```diff
@@ -2075,12 +2075,10 @@ public int numDocs(Query a, DocSet b) throws IOException {
       return a == absQ ? b.intersectionSize(positiveA) : b.andNotSize(positiveA);
     } else {
       // If there isn't a cache, then do a single filtered query
-      // NOTE: we cannot use FilteredQuery, because BitDocSet assumes it will never
```

Comment:
obsolete comment; FilteredQuery hasn't existed in quite some time, and even then I'm a bit confused by the comment as this doesn't use UninvertedField so who cares.